### PR TITLE
Ui improvements

### PR DESCRIPTION
**Pull request by skkr**
@AugustoL please review.

### CODE REVIEW

**Comment by skkr** [comment]
@AugustoL please review the latest changes. I think I'll probably finish the Explorer section on monday, and will start to work on the mobile version right after.

File: app/views/Hotel.js [hunk 1]
```diff
@@ -397,13 +398,14 @@ export default class App extends React.Component {
...
-            <li><button class={"btn btn-default btn-action" + (self.state.section == 'hotelBookings' ? " btn-success" : "")} onClick={() => self.setState({section: 'hotelBookings'})}>Hotel Bookings</button> </li>
-            <li><button class={"btn btn-default btn-action" + (self.state.section == 'hotelTxs' ? " btn-success" : "")} onClick={() => self.setState({section: 'hotelTxs'})}>Blockchain Txs</button> </li>
-          </ul>
-        </div>
+      <div class="nav flex-column nav-pills" id="actions" role="tablist" aria-orientation="vertical">
+        <a class={'nav-link ' + (self.state.section == 'hotels' ? 'active bg-secondary' : '')} id="v-pills-home-tab" data-toggle="pill" role="tab" aria-controls="v-pills-home" aria-selected="true"
+          onClick={() => self.setState({section: 'hotels'})}>Home</a>
+        <a class={'nav-link ' + (self.state.section == 'hotelBookings' ? 'active bg-secondary' : '')} id="v-pills-profile-tab" data-toggle="pill" role="tab" aria-controls="v-pills-profile" aria-selected="false"
+          onClick={() => self.setState({section: 'hotelBookings'})}>Profile</a>
```
Comment (adklempner) [inline app/views/Hotel.js]: This link should display Bookings, not Profile. And Messages -> Transactions

**Comment by skkr** [comment]
@AugustoL, just pushed the final adjustments on the sections you asked me to finish today, including the changes on the hotel manager menu. I'll continue working to make the design responsive. Shoud I keep working on this branch?

**Comment by AugustoL** [comment]
Thanks @skkr ! Im going to review it tomorrow.

For the responsive design please continue working on another branch and then we will rebase from master once we have the new design on it.

**Comment by skkr** [comment]
Copy that, thank for the quick feedback!

**Pull request merged**